flexible/java/samples/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,31 @@
+runtime: java
+env: flex
+
+handlers:
+- url: /.*
+  script: this field is required, but ignored
+
+use_memcache_proxy: true
+
+# [START env_variables]
+env_variables:
```

Comment:
This is probably how it's going to stay, but it's nice if we can do variable substitution `${blah}`, but I don't see a way to support gradle with that.